sagemaker-training-compiler/tensorflow/multiple_gpu_single_node/vision-transformer.ipynb

```diff
@@ -0,0 +1,1080 @@
+{
```

Comment:
<p>memory of a <code>ml.p3.8xlarge</code> instance -&gt; memory of the V100 GPUs in a <code>ml.p3.8xlarge</code> instance</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>